src/Tools/Source/CompilerGeneratorTools/Source/IOperationGenerator/IOperationClassWriter.cs

```diff
@@ -298,11 +303,56 @@ void writeEnumElement(string kind, int value, string operationName, string? extr
             }
         }
 
-        private string GetKindName(string operationName) => operationName[1..^9];
+        private string GetBaseName(string operationName) => operationName[1..^9];
+
+        private void WriteVisitors()
+        {
+            WriteLine(@"public abstract partial class OperationVisitor
```

Comment:
nit: can you put the string on the next line.  i think it makes things easier to read (esp. on github where it's harder to tell what is code and what is a literal).

Reply:
No, I can't. It will start by putting whitespace for the current line in, which will result in a blank line with a bunch of spaces in it.